### ISSUE TITLE
New scoring method and new payload in win query

### ISSUE THREAD

**Issue by DooGer1** [Galkurta/Coinsweeper-BOT#7]
scoring method as in the game in screenshots: 
const score = Math.floor((10 * i + Math.max(0, 1200 - 10 * game_time) + 2000) * (1 + 9 / 54)) / 10 + value(this.game.id);

<img width="624" alt="Снимок экрана 2024-09-26 в 15 56 56" src="https://github.com/user-attachments/assets/66059a9f-da27-4b2a-8d45-95a50dfb3759">
<img width="599" alt="Снимок экрана 2024-09-26 в 15 57 47" src="https://github.com/user-attachments/assets/914b62f4-d0f7-4f9f-877f-2bafa69ccf61">

and a new payload in the win: request

const i = '66f28527e5c5186a622a58a2v$2f1';
        const score = Math.floor((10 * i + Math.max(0, 1200 - 10 * game_time) + 2000) * (1 + 9 / 54)) / 10 + value(this.game.id);
        const mr = `${this.game.id}-${new Date(this.game.createdAt).getTime()}`;

        const xr = i + "-" + mr;
        const kr = `${Bt}-${er.id}`;
        const h = CryptoJS.HmacSHA256(kr, xr).toString(CryptoJS.enc.Hex);

<img width="580" alt="Снимок экрана 2024-09-26 в 15 59 35" src="https://github.com/user-attachments/assets/3f167b12-c71a-4eb6-8a3a-de3265a53e2f">

and another suggestion is to remove the authorization request and put tokens in the file

**Comment by Galkurta** [comment]
feel free to make a pull request sir

**Comment by gitnnm** [comment]
> feel free to make a pull request sir, i'm very helpful and very grateful if there is someone who makes improvements

`Too Many Requests, Please Wait`

**Comment by DooGer1** [comment]
Yes, now the bot has been updated, we need to look for a new approach
